pyproject.toml

```diff
@@ -0,0 +1,100 @@
+[project]
+name = "movement"
+authors = [
```

Comment:
I would only list authors as they contribute significantly to the project. I'm all for sharing credit, but these emails are the ones listed on PyPI, and I would expect that person to be one of the core maintainers. Contributions are listed on the GH page anyway.

Reply:
Undetstood. So for now just list myself and add other later, as required?

Reply:
Yeah I think so. Anyone who can answer questions about it I think.